test/martha/martha_v3.test.js

```diff
@@ -728,6 +724,83 @@ test.serial('martha_v3 parses HCA response correctly', async (t) => {
     sinon.assert.callCount(getJsonFromApiStub, 1);
 });
 
+const testWithTimeout = (ms, asyncTestFn) => (t) => {
+    const waitThenFail = async () => {
+        await delay(ms);
+        t.fail(`Test did not complete within ${ms}ms`);
+    };
+    return Promise.race([asyncTestFn(t), waitThenFail()]);
+};
+
+test.serial('martha_v3 succeeds within 60 seconds even if fetching a signed URL never returns', testWithTimeout(5 * 1000, async (t) => {
+    overridePencilsDownTime(3);
```

Comment:
ToL: Is the "60 seconds" still applicable with the override? Perhaps drop it, leaving "succeeds even if fetching..."

Reply:
Oh yeah, I noticed that at one point and then forgot to get back to it. I'll update the test name and probably add a clarifying comment.